actors/abi/sector.go

```diff
@@ -64,140 +64,166 @@ func NewStoragePower(n int64) StoragePower {
-		return RegisteredProof_StackedDRG64GiBWindowPoSt, nil
-	case RegisteredProof_StackedDRG32GiBSeal, RegisteredProof_StackedDRG32GiBWinningPoSt, RegisteredProof_StackedDRG32GiBWindowPoSt:
-		return RegisteredProof_StackedDRG32GiBWindowPoSt, nil
-	case RegisteredProof_StackedDRG2KiBSeal, RegisteredProof_StackedDRG2KiBWinningPoSt, RegisteredProof_StackedDRG2KiBWindowPoSt:
-		return RegisteredProof_StackedDRG2KiBWindowPoSt, nil
-	case RegisteredProof_StackedDRG8MiBSeal, RegisteredProof_StackedDRG8MiBWinningPoSt, RegisteredProof_StackedDRG8MiBWindowPoSt:
-		return RegisteredProof_StackedDRG8MiBWindowPoSt, nil
-	case RegisteredProof_StackedDRG512MiBSeal, RegisteredProof_StackedDRG512MiBWinningPoSt, RegisteredProof_StackedDRG512MiBWindowPoSt:
-		return RegisteredProof_StackedDRG512MiBWindowPoSt, nil
+	case RegisteredSealProof_StackedDrg64GiBV1:
+		return RegisteredPoStProof_StackedDrgWindow64GiBV1, nil
+	case RegisteredSealProof_StackedDrg32GiBV1:
+		return RegisteredPoStProof_StackedDrgWindow32GiBV1, nil
+	case RegisteredSealProof_StackedDrg2KiBV1:
+		return RegisteredPoStProof_StackedDrgWindow2KiBV1, nil
+	case RegisteredSealProof_StackedDrg8MiBV1:
+		return RegisteredPoStProof_StackedDrgWindow8MiBV1, nil
+	case RegisteredSealProof_StackedDrg512MiBV1:
+		return RegisteredPoStProof_StackedDrgWindow512MiBV1, nil
 	default:
 		return 0, errors.Errorf("unsupported mapping from %+v to PoSt-specific RegisteredProof", p)
 	}
 }
 
-// RegisteredSealProof produces the seal-specific RegisteredProof corresponding
+// RegisteredWindowPoStProof produces the PoSt-specific RegisteredProof corresponding
 // to the receiving RegisteredProof.
-func (p RegisteredProof) RegisteredSealProof() (RegisteredProof, error) {
-	switch p {
-	case RegisteredProof_StackedDRG64GiBSeal, RegisteredProof_StackedDRG64GiBWindowPoSt, RegisteredProof_StackedDRG64GiBWinningPoSt:
-		return RegisteredProof_StackedDRG64GiBSeal, nil
-	case RegisteredProof_StackedDRG32GiBSeal, RegisteredProof_StackedDRG32GiBPoSt, RegisteredProof_StackedDRG32GiBWindowPoSt, RegisteredProof_StackedDRG32GiBWinningPoSt:
-		return RegisteredProof_StackedDRG32GiBSeal, nil
-	case RegisteredProof_StackedDRG2KiBSeal, RegisteredProof_StackedDRG2KiBPoSt, RegisteredProof_StackedDRG2KiBWindowPoSt, RegisteredProof_StackedDRG2KiBWinningPoSt:
-		return RegisteredProof_StackedDRG2KiBSeal, nil
-	case RegisteredProof_StackedDRG8MiBSeal, RegisteredProof_StackedDRG8MiBPoSt, RegisteredProof_StackedDRG8MiBWindowPoSt, RegisteredProof_StackedDRG8MiBWinningPoSt:
-		return RegisteredProof_StackedDRG8MiBSeal, nil
-	case RegisteredProof_StackedDRG512MiBSeal, RegisteredProof_StackedDRG512MiBPoSt, RegisteredProof_StackedDRG512MiBWindowPoSt, RegisteredProof_StackedDRG512MiBWinningPoSt:
-		return RegisteredProof_StackedDRG512MiBSeal, nil
-	default:
-		return 0, errors.Errorf("unsupported mapping from %+v to seal-specific RegisteredProof", p)
+func (p RegisteredPoStProof) RegisteredWindowPoStProof() (RegisteredPoStProof, error) {
```

Comment:
Similarly I don't think we need this one, if we have the SealProof->WindowPoStProof one.

Reply:
Done.